src/lib/__tests__/index.ts

```diff
@@ -108,3 +108,10 @@ test('setExpectations with a different baseUrl for the stub server', () => {
     body: JSON.stringify(mockStub)
   });
 });
+
+test('setSpecmaticEnvVariable updates the environment variable value in the specmatic.json file', () => {
+  const name = "key"
+  const updatevalue = "updated-value"
+  const fileName = "specmatic.json"
+  expect(setSpecmaticEnvVariable(name, updatevalue, fileName))
```

Comment:
What are we expecting here?
A good test would be to add another assertion here, where you read the file and make sure the value has changed from "initial-value" to "updated-value"

Reply:
Refactored the test and expecting the boolean value based on the status if updated or not

Reply:
How does the boolean ensure that the correct thing was updated or not? I might be setting "updated-value-1", but due to a bug in my method in actual the file is setting "updated-value-2". Hence we should actually read the file and assert on its contents

Reply:
Added a new function to check if the required file is updated or not